mgr/cephadm: Adding --storage.tsdb.retention.size prometheus option (#48006)
ceph/ceph: doc/cephadm/services/monitoring.rst

> Setting Prometheus Retention Time

Reviewer: ```suggestion
Setting Prometheus Retention Time and Size
```
I think we ought to mention both in the section title

Author: fixed

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Cephadm provides the option to set the Prometheus TDSB retention time specifying
> ``retention_time`` and/or ``retention_size`` fields in the Prometheus service spec.

Reviewer: ```suggestion
Cephadm provides the option to set the Prometheus TDSB retention time and/or size by specifying
the ``retention_time`` and``retention_size`` fields respectively in the Prometheus service spec.
```
Personal opinion on wording. Feel free to just mark this one resolved if you disagree.

Reviewer 2: I'd suggest something more concise like

 "Cephadm can configure Prometheus TSDB retention by specifying ``retention_time`` and``retention_size`` values in the Prometheus service spec."

Reviewer: > I'd suggest something more concise like
> 
> "Cephadm can configure Prometheus TSDB retention by specifying `retention_time` and`retention_size` values in the Prometheus service spec."

I like that better as well. @rkachach if you're going to change I'd say go with @anthonyeleven suggestion over mine.

Author: fixed

> The retention time value defaults to 15 days (15d). User can set different value/unit where

Reviewer: ```suggestion
The retention time value defaults to 15 days (15d). Users can set a different value/unit where
```

Author: fixed
